models/migrations/migrations.go

```diff
@@ -294,8 +294,10 @@ var migrations = []Migration{
 	NewMigration("Add commit id and stale to reviews", addReviewCommitAndStale),
 	// v119 -> v120
 	NewMigration("Fix migrated repositories' git service type", fixMigratedRepositoryServiceType),
-	// v120 -> v121
+	// v120 - v121
```

Comment:
merge confilct resolve issue
```suggestion
	// v120 -> v121
```